test/unit/org/apache/cassandra/tcm/ClusterMetadataSerializerTest.java

```diff
@@ -47,16 +47,16 @@ public class ClusterMetadataSerializerTest
     public void serdeLatest()
     {
         DataOutputBuffer output = new DataOutputBuffer();
-        qt().forAll(new ClusterMetadataBuilder().build()).checkAssert(orFail(cm -> {
+        qt().forAll(Generators.toGen(new ClusterMetadataBuilder().build())).check(cm -> {
             AsymmetricMetadataSerializers.testSerde(output, ClusterMetadata.serializer, cm, NodeVersion.CURRENT_METADATA_VERSION);
-        }));
+        });
     }
 
     @Test
     public void serdeWithoutAccord()
     {
         DataOutputBuffer output = new DataOutputBuffer();
-        Gen<ClusterMetadata> gen = new ClusterMetadataBuilder().build().assuming(cm -> {
+        Gen<ClusterMetadata> gen = Generators.toGen(new ClusterMetadataBuilder().build()).filter(cm -> {
```

Comment:
accord `filter` doesn't have a bounded amount of retries but `accuming` with QuickTheories does, making this test flakey.  The issue is when a CM is generated without anything accord related... we filter those out... if you get unlucky and do this many times in a row `assuming` would reject the build; `filter` does not